CHANGELOG.md

```diff
@@ -7,7 +7,76 @@ documentation before upgrading to a new release.
 
 Released closed milestones can be found on [GitHub](https://github.com/Icinga/icinga2/milestones?state=closed).
 
-# 2.12.6 (2021-08-19)
+## 2.12.7 (2022-04-14)
+
+This version includes bugfixes for many features of Icinga 2, including fixes for multiple crashes. It also includes a
+number of fixes and improvements for Icinga DB.
```

Comment:
Which ones exactly?

Reply:
Well, maybe I should extend that sentence with "if you upgrade to 2.13.3 instead" :see_no_evil: